titan/params/partnership.yml

```diff
@@ -116,25 +105,29 @@ partnership:
           default: 0.0138
   injection:
     frequency:
-      distribution:
-        type: enum
-        class: distributions
-        default: poisson
-      vars:
-        type: bin
-        description: Pass values in order of bin keys to the distriubtion, starting at 1
-        fields:
-          value:
-            type: float
-          value_type:
-            type: enum
-            values:
-              - int
-              - float
-        default:
-          1:
-            value: 0
-            value_type: int
+      type: sub-dict
```

Comment:
should frequency be moved up a level, the it sub-dict on acts_allowed and bond_types?

Reply:
I'm hesitant on this one since not all interaction types use this format for determining frequency (Inj uses demographics)

Reply:
I think this one is deleted in my branch - is it being used somewhere on this branch?

Reply:
No, just not deleted on this branch.